docs/platform/settings/external.services.md

```diff
@@ -0,0 +1,33 @@
+!!! warning "While Error Reporting and Statistics data is anonymous and does *not* contain any personal information, Developer List and Settings Synchronize data are *not* anonymous and may contain personally identifiable information."
+!!! note "This information is collected for use only by [Consulo.io] and will never be transmitted to any third party."
+
+This is where you manage your [Consulo.io] services.
+
+The [Consulo.io] account you are signed into is shown at the top of the window.
+
+## Services
+
+- Error Reporting - **Default: Enabled** (Anonymous)<br/>
+	Whether to send anonymous error reporting data to [Consulo.io].<br/>
+	This helps the developer(s) maintain the IDE and fix these errors.<br/>
+	This data **is** anonymous.
+- Developer List (Error Reporting) - **Default: Disabled**<br/>
+	Whether to attach a list of developers to your error reporting data when it is sent to [Consulo.io].<br/>
+	<!--- Why? ---><br/>
```

Comment:
Why is the developer list asked for?
This would be nice to tell users so they can properly judge whether or not they want to send the data.

Reply:
It's API check for calling develop list from hub. It's only allow if user is already developer. I just show this settings for all - but maybe it will be good, hide it, and show only for developer auth.

It's used to assign developer inside Error Reporter dialog. For example if developer catch exception, and want assign it to another (it can be viewed inside hub, if attached)